app/src/app/rc.js

```diff
@@ -0,0 +1,27 @@
+import path from 'path'
+import {app} from 'electron'
+import {existsSync} from 'fs'
+import loadJson from 'load-json-file'
+import {isWindows, isOSX} from './utils'
+
+// Directory where of .app or .exe file lives.
+let dir
+
+if (isOSX()) dir = path.normalize(app.getAppPath() + '/../../../..')
```

Comment:
What is this supposed to do? In the issue I read that for OSX we want to use the user home dir, if that's the case then you could use `process.env.HOME`